### PR TITLE
Some minor static code analysis suggested changes.

### PR DESCRIPTION
**Pull request by rsuller**

### CODE REVIEW

File: conf/env.conf [hunk 1]
```diff
@@ -1,12 +1,25 @@
-testing {
-chips_url=
-username=
-password=
-chips-jdbc=
-chips-db-user=
-chips-db-pass=
-chips-rest-api = 
-ef-submission-api = 
+kermit {
```
Comment (danfalc) [inline conf/env.conf:1]: Don't want this in there do we?

Comment (rsuller) [reply conf/env.conf:1]: Ideally no. But this is causing us so much of an issue I am at the point of just thinking, leave it be. it is no more exposed than what we already have.

Comment (danfalc) [reply conf/env.conf:1]: I don't mind either way. Add in the Chips rest and ef submission api for kermit then and lets get it in

Comment (danfalc) [reply conf/env.conf:1]: Or will that get us in trouble?

Comment (rsuller) [reply conf/env.conf:1]: That information is already available, so I think add it in 🤷‍♂️

Comment (danfalc) [reply conf/env.conf:1]: Go for it then

Comment (rsuller) [reply conf/env.conf:1]: Done

**Review by danfalc — APPROVED** [review]

**Pull request merged**